src/main/java/at/hannibal2/skyhanni/config/Storage.java

```diff
@@ -445,6 +450,34 @@ public static class DungeonStorage {
 
             @Expose
             public Map<DungeonAPI.DungeonFloor, Integer> bosses = new HashMap<>();
+
+            @Expose
+            public List<DungeonRunInfo> runs = Stream.generate(DungeonRunInfo::new).limit(CroesusChestTracker.Companion.getMaxChests()).collect(Collectors.toCollection(ArrayList::new));
+
+
+            public static class DungeonRunInfo {
+
+                public DungeonRunInfo() {
```

Comment:
do we need this empty default condstructor?

Reply:
yes, it is used to setup the list in storage